packages/components/psammead-story-promo-list/src/index.stories.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { Fragment } from 'react';
+import { storiesOf } from '@storybook/react';
+import nanoid from 'nanoid';
```

Comment:
As its not in prod, nanoid is fine here, but it needs to be inc in the package.json :)

Reply:
for context, we are moving away from nanoid due to the generated ID being different in simorgh server/client renders, but thats not a problem here :)

Reply:
Although `nanoid` is fine, shall I use another `key`?

Reply:
You could use something like the headline yeah